brianlong/www.validators.app
# 186664127-avatar-gif-support

#### What's this PR do?
- adds rmagick library
- crops and saves gifs to correct size as validator avatars

#### How should this be manually tested?
- run test
- change avatar_url for one of the validators to gif or find validator that already has animated avatar
- run app/services/update_avatar_file_service.rb for this validator and confirm the gif was converted properly

#### What are the relevant tickets?
- [URL to the PivotalTracker ticket](https://www.pivotaltracker.com/story/show/186664127)

#### Task completed checklist:
- [] Is there appropriate test coverage?
- [] Did I take into consideration possible security vulnerabilities?
- [] Are the controllers secure?
- [] Was every important matter documented?

#### Things to watch out for
- [N] Does this PR have any migrations?
- [N] Does this PR add new dependencies?
